trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -442,11 +442,21 @@ def build_surrogate(self):
 
         for jj in range(len(AoA_points)):
             for ii in range(len(mach_points)):
-                CM_sur[ii,jj]    = cm_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
-                Cm_a_sur[ii,jj]  = cm_alpha_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
-                Cn_b_sur[ii,jj]  = cn_beta_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
-                NP_sur[ii,jj]    = neutral_point_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))                
+                CM_sur[ii,jj]    = cm_surrogate.predict([np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]])])
+                Cm_a_sur[ii,jj]  = cm_alpha_surrogate.predict([np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]])])
```

Comment:
What is this?

Reply:
May have been a Git Merge issue. I'll take out